src/rust/ide/view/graph-editor/src/builtin/visualization/java_script/mapView.js

```diff
@@ -68,29 +74,131 @@ class MapViewVisualization extends Visualization {
         }
 
         let defaultMapStyle = 'mapbox://styles/mapbox/light-v9';
+        let accentColor     = [1,234,146];
         if (document.getElementById("root").classList.contains("dark")){
             defaultMapStyle = 'mapbox://styles/mapbox/dark-v9';
+            accentColor     = [222,162,47];
         }
 
+        let preparedDataPoints = []
+        let computed           = this.prepareDataPoints(parsedData,preparedDataPoints,accentColor);
+
+        const scatterplotLayer = new deck.ScatterplotLayer({
+            data: preparedDataPoints,
+            getFillColor: d => d.color,
+            getRadius: d => d.radius
+        })
+        //
+        let latitudeMatch  = parsedData.latitude !== undefined && parsedData.latitude !== null;
```

Comment:
This is error prone. You sohuld always check for both `undefined` and `null` here. I'd create utility for that:

```js
function ok(t) {
    if t !== undefined && t !== null
}
```